solution-base/mongodb/charts/mongodb-sharded/templates/mongos/mongos-service.yaml

```diff
@@ -19,6 +19,9 @@ spec:
   {{- if and (eq .Values.service.type "ClusterIP") .Values.service.clusterIP }}
   clusterIP: {{ .Values.service.clusterIP }}
   {{- end }}
+  {{- if .Values.service.externalIPs }}
+  externalIPs: {{- toYaml .Values.service.externalIPs | nindent 4 }}
+  {{- end }}
```

Comment:
I might be missing the context, so please help me
When would we need specific external IP addresses

Reply:
I think we don't need them today, it's a mapper that links to any configuration we might need for this purpose, coming from the Helm charts, but that doesn't mean we use any